instrumentation/cassandra/cassandra-3.0/javaagent/src/test/java/CassandraClientTest.java

```diff
@@ -57,10 +61,17 @@ static void beforeAll() {
             .withStartupTimeout(Duration.ofMinutes(2));
     cassandra.start();
 
+    cassandraHost = cassandra.getHost();
+    try {
+      cassandraIp = java.net.InetAddress.getByName(cassandra.getHost()).getHostAddress();
+    } catch (UnknownHostException e) {
+      // Default to 127.0.0.1
+      cassandraIp = "127.0.0.1";
+    }
```

Comment:
under what condition is `UnknownHostException` thrown?

Reply:
Sure, I can make that change. I'll have it pushed in a few minutes.

Reply:
@trask One of the places this exception is thrown is the constructor for `RocketMqProxyContainer`. I've never seen a constructor with a `throws` before. If I add it, IntelliJ's recommendation for the instantiation is to wrap _it_ in a try/catch instead:
```java
  private static final RocketMqProxyContainer container;

  static {
    try {
      container = new RocketMqProxyContainer();
    } catch (UnknownHostException e) {
      throw new RuntimeException(e);
    }
  }
```

What is your recommendation here? Keeping the original default in the constructor or adding the try/catch around the instantiation?

Reply:
I went with wrapping the instantiation in a try/catch, but changing RuntimeException to IllegalStateException to satisfy the ThrowSpecificExceptions requirement.